mindmeld/components/dialogue.py

```diff
@@ -906,6 +909,18 @@ def _end_slot_fill(self, request, responder, async_mode):
             params=request.params,
         )
 
+        if self._invoked:
+            kwargs = {'domain': request.domain, 'intent': request.intent}
+            name = self._entrance_handler.__name__
+            if self._previous_rule:
+                try:
```

Comment:
Could you add a comment here saying why we do this try/exception block. It would be useful for maintaining this code.

Reply:
We should also update the function doc